config/application.rb

```diff
@@ -14,7 +14,8 @@
 module IdentityPki
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.1
```

Comment:
Pretty far behind here, but nothing in the defaults jumps out: https://github.com/rails/rails/blob/v6.1.7.3/railties/lib/rails/application/configuration.rb#L113-L207